qa/acceptance-tests/src/test/java/io/camunda/it/rdbms/db/group/GroupIT.java

```diff
@@ -154,6 +156,7 @@ public void shouldFindWithFullFilter(final CamundaRdbmsTestApplication testAppli
     assertThat(searchResult.items().getFirst().groupKey()).isEqualTo(group.groupKey());
   }
 
+  @Disabled("I think this test is not valid anymore as now the key of the table is a String")
```

Comment:
💭 This test is not valid anymore, as now we are using a String as primary key. We can't properly query for after or before as the order can't be ensured as before

Reply:
@nicpuppa can't we just include the `groupId` on [line 185](https://github.com/camunda/camunda/pull/31015/files#diff-28aa36ea486e69ed83cf498a48c5f7b4f5e14455ea62c92f0d7eecbe9c61ab7aR185), similar to how it's done for the `TenantIT` test [here](https://github.com/camunda/camunda/blob/5d87780b41799fa60c84fcafa941965840162261/qa/acceptance-tests/src/test/java/io/camunda/it/rdbms/db/tenant/TenantIT.java#L186-L188)?

Reply:
Somehow I thought that string comparison wasn't working as expected, but the test succeeds so I re-enable it @koevskinikola